kevlar/varmap.py

```diff
@@ -45,9 +45,14 @@ def __init__(self, contig, cutout, score=None, cigar=None, strand=1,
         self.matedist = None
         self.vartype = None
         self.alnmatch = None
-        for pattern, vartype in patterns.items():
+        for pattern, (vartype, rightcheck) in patterns.items():
             matchobj = re.match(pattern, cigar)
             if matchobj:
+                if rightcheck:
+                    idx = 6 if vartype == 'snv' else 9
+                    rightmatchlen = int(matchobj.group(idx))
+                    if rightmatchlen > 5:
+                        continue
```

Comment:
This check must've been lost in recent refactoring. This is a gnarly stopgap, but it gets 'r done for now.